test/functional/apps/discover/_discover.js

```diff
@@ -1,15 +1,14 @@
+
+import expect from 'expect.js';
+
 import {
   bdd,
   scenarioManager,
-  common,
-  discoverPage,
-  settingsPage,
-  headerPage,
   esClient,
   elasticDump
 } from '../../../support';
 
-var expect = require('expect.js');
+import PageObjects from '../../../support/page_objects';
```

Comment:
Page Object` is a normal term for this (see for example http://martinfowler.com/bliki/PageObject.html). (Of course, it doesn't have to be named `Page object`, though)